content/docs/command-reference/remote/modify.md

```diff
@@ -288,23 +288,24 @@ For more information about the variables DVC supports, please visit
 
 <details>
 
-### Click for S3 API compatible storage
+### Click for S3-compatible storage
 
-To communicate with a remote object storage that supports an S3 compatible API
-(e.g. [Minio](https://min.io/),
-[DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
-[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.),
-configure the remote's `endpointurl` explicitly:
+To communicate with a remote object storage that supports an S3-compatible API,
+configure the remote's `endpointurl`:
 
 ```dvc
 $ dvc remote add -d myremote s3://mybucket/path
 $ dvc remote modify myremote endpointurl \
                     https://object-storage.example.com
 ```
 
-Besides that, any parameters that are available for Amazon S3 (see previous
-section) may be available for S3 compatible storage. For example, let's setup a
-DVC remote using the `example-name`
+It's also important to setup appropriate authentication with any of the related
+parameters available for regular S3 storage (see previous section). Otherwise,
+DVC may try to use default AWS credentials (if any), which will probably cause
+an error.
```

Comment:
Resubmitting https://github.com/iterative/dvc.org/pull/2415#discussion_r619635479:
```suggestion
It's also important to setup appropriate authentication with any of the related
parameters available for regular S3 storage (see previous section). Otherwise,
DVC may try to use default AWS credentials (if any), which will probably cause
an error.
```
Cc @iesahin

Reply:
BTW no problem if the format isn't perfect, Restyled can fix it later. Committing...